freqtrade/exchange/exchange.py

```diff
@@ -74,6 +75,12 @@ class Exchange:
     }
     _ft_has: Dict = {}
 
+    _leverage_brackets: Dict = {}
```

Comment:
You'll need to be careful with class-level variables (your just added patch is hiding a underlying problem, so please remove it to avoid problems later).

Creating the variable here makes it a class-variable, not an instance variable.
As it's a mutable type (dict) - you'll need to be careful how to mutate it, as it can become kindof a singleton (depending how you use it - and terminology is probably off,  as a singleton is really a class that can only exist once).

While you could "reset" it in `__init__` (reassign it) - there's little point for that, so you could simply move this line there and have it be `self._leverage_brackets: Dict = {}`.

Now depending how you're using it, it'll be persisted across tests or not.

as the code works slightly different for binance and kraken, this failure is not apparent in the kraken test.
* in Kraken's case, you're assigning the variable directly (`self._leverage_brackets = ...`) and therefore converting it a instance variable (there'll still be an `Exchange._leverage_brackets` around that is an empty dict).
* in Binance's case, you're using it as `self._leverage_brackets[pair] =` - mutating the existing class-level object (it's the same when used as `self._leverage_brackets` and `Exchange._leverage_brackets`).

As you're mutating the class-level object, it's then also available for FTX (and potentially any other exhange/test that comes later), as the pytest session runs all in the same python interpreter (it's not killing the interpreter, so class-level states / instances will be retained.

I'd propose a very simple fix in moving the creation of the empty dict to `__init__`  (making it an explicit instance-level property) - removing the patchy hack to fix the test failure (the failure was good, as it showed this flaw).

Reply:
Is there a technical term to describe this behavior?